main.js

```diff
@@ -160,4 +160,249 @@ const inventory = [
     originalStock: 10,
     sold: 8,
   },
-];
\ No newline at end of file
+];
+
+// **----------  OPDRACHT 1A ----------**
+
+const typeTvAndNames = inventory.map ((stockTv)=>{
+  return stockTv.type +" "+ stockTv.name;
```

Comment:
Ik vind dit altijd wat beter te lezen
```suggestion
  return `${stockTv.type} ${stockTv.name}`;
```